Lib/gftools/builder/__init__.py

```diff
@@ -642,17 +643,28 @@ def main(args=None):
 
     args = parser.parse_args(args)
 
+    from gftools.builder._ninja import NinjaBuilder
+    builder_class = GFBuilder
+
+    try:
+        if platform.system() != "Windows":
+            builder_class = NinjaBuilder
+    except ImportError as e:
```

Comment:
why this `except ImportError` here? I don't think `platform.system()` can ever fail with ImportError...

Reply:
or you wanted to move `from gftools.builder._ninja import NinjaBuilder` line inside the try/except block maybe

Reply:
Argh, yeah, I did.